test/Sema/enum_conformance_synthesis.swift

```diff
@@ -23,6 +24,7 @@ enum Generic<T> {
 
 if Generic<Foo>.A == .B { }
 var gaHash: Int = Generic<Foo>.A.hashValue
+print(Generic<Foo>.allCases)
```

Comment:
Is this left over from debugging or intended to be included in this file? I suppose it doesn't matter since we're `-typecheck`ing only.

Reply:
To suppress an unused lvalue warning

Reply:
I think `_ =` would be (mildly) preferable